src/background/messaging/rpc-methods/stx-transfer-sip10-ft.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  type ClarityValue,
+  createAddress,
+  noneCV,
+  postConditionToWire,
+  serializeCV,
+  serializePostConditionWire,
+  standardPrincipalCVFromAddress,
+  uintCV,
+} from '@stacks/transactions';
+import { createUnsecuredToken } from 'jsontokens';
+
+import { type RpcParams, type RpcRequest, stxTransferSip10Ft } from '@leather.io/rpc';
+import { TransactionTypes, getStacksAssetStringParts } from '@leather.io/stacks';
+
+import { RouteUrls } from '@shared/route-urls';
+import { makeFtPostCondition } from '@shared/utils/post-conditions';
+
+import { handleRpcMessage } from '../handle-rpc-message';
+import {
+  type RequestParams,
+  getStxDefaultMessageParamsToTransactionRequest,
+  validateRequestParams,
+} from '../messaging-utils';
+
+async function getMessageParamsToTransactionRequest(params: RpcParams<typeof stxTransferSip10Ft>) {
+  const { contractAddress, contractAssetName, contractName } = getStacksAssetStringParts(
+    params.asset
+  );
+  const result = await chrome.storage.local.get('stacksAddress');
```

Comment:
Why not persist this address in redux, and just read it directly from the store versus adding new state?